src/substrate/types.ts

```diff
@@ -158,6 +161,17 @@ export interface ISomeOffline extends IEvent {
   validators: Array<AccountId>;
 }
 
+/**
+ * Offences Events
+ */
+export interface IOffence extends IEvent {
+  kind: EventKind.Offence;
+  offenceKind: string;
+  opaqueTimeSlot: string;
+  applied: boolean;
+  offenders: Array<ValidatorId>
```

Comment:
`ValidatorId` is a polkadot type, so we should replace this with `string`, or if we wanted to be consistent, use the `AccountId` type I defined above (which is just a string).